bezier_library/doc/README_bezier_grinding_surfacing.md

```diff
@@ -38,109 +38,113 @@ This process is used to create extrication passes, in order to avoid collisions
 
 ![harmonize_line_orientation](harmonize_line_orientation.gif)
 
-16. Filter extrication trajectory: Extrication
+17. Filter extrication trajectory: Extrication
 ---
 [filterExtricationTrajectory](README_filter_extrication_trajectory.md) filters the extrications trajectories. It allows to avoid having poses which are inaccessible for the robot.
 
 ![extrication_filter](extrication_filter.gif)
 
-17. Estimate slicing plane: Last extrication pass
+18. Estimate slicing plane: Last extrication pass
 ---
 [estimateExtricationSlicingPlane](README_estimate_extrication_slicing_plane.md) computes the plane equation needed to generate the last extrication line which goes from the last point of the last grinding line to the first point of the first grinding line.
 
 ![estimate_extrication_slicing_plane](estimate_extrication_slicing_plane.png)
 
-18. Slice mesh with planes: Last extrication pass
+19. Slice mesh with planes: Last extrication pass
 ---
 [sliceMeshWithPlanes](README_slice_mesh_with_planes.md) calls the function `sliceMeshWithPlane` to cut the mesh. This operation generates the last extrication line.
 
 ![slice_mesh_with_planes_last_extrication](slice_mesh_with_planes_last_extrication.png)
 
-19. Generate robot poses along stripper: Last extrication pass
+20. Generate robot poses along stripper: Last extrication pass
 ---
 [generateRobotPosesAlongStripper](README_generate_robot_poses_along_stripper.md) generates the last extrication trajectory along a line. The trajectory is a succession of poses represented by an orientation and a position in the 3D space.
 
 ![generate_last_extrication_lines](generate_last_extrication_lines.png)
 
-20. Harmonize line orientation: Last extrication pass
+21. Harmonize line orientation: Last extrication pass
 ---
 [harmonizeLineOrientation](README_harmonize_line_orientation.md) checks that the last extrication line is correctly oriented. It acheive that by comparing the orientation of the line to a vector of reference which goes from the last point of the last grinding line to the first point of the 
 first grinding line.
 
 ![harmonize_line_orientation_last_extrication](harmonize_line_orientation_last_extrication.gif)
 
-21. Filter extrication trajectory: Last extrication pass
+22. Filter extrication trajectory: Last extrication pass
 ---
 [filterExtricationTrajectory](README_filter_extrication_trajectory.md) filters the last extrication trajectory. It allows to avoid having poses which are inaccessible for the robot.
 
 ![extrication_filter](last_extrication_filter.png)
 
-22. Invert X axis of poses: Last extrication pass
+23. Invert X axis of poses: Last extrication pass
 ---
 [invertXAxisOfPoses](README_invert_x_axis_of_poses.md) reverse the x axe of the last extrication trajectory poses thus avoiding the tools to make a PI rotation on himself to reach the initial start point.
 
-23. Result
+24. Result
```

Comment:
This is hell, that's why we should switch to  a "smarter" markup language in the future :)